difflint/lib.py

```diff
@@ -249,7 +197,25 @@ def detect_new_diff_lint_errors(log_output):
 
 
 def main():
-    if not lint_script_exists() or not all_files_configured():
+    parser = argparse.ArgumentParser(description='Linter that will examine ' +
+        'only new changes as you commit them.')
+    parser.add_argument('-c', '--check', action='store_true',
```

Comment:
Being able to have the `difflint` program not fail if some programs are missing. This isn't set up quite consistently right now, since `difflint` is hardcoded not to fail if programs are missing. But ideally you could put something like this in your git hook:
```sh
if difflint --check; then
    difflint $@
fi
```